editor/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -2,14 +2,15 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
-import { unescape as unescapeString, without, groupBy, map, repeat, find } from 'lodash';
+import { unescape as unescapeString, without, map, repeat, find } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { __, _x } from '@wordpress/i18n';
 import { Component, compose } from '@wordpress/element';
 import { withInstanceId, withSpokenMessages } from '@wordpress/components';
+import { buildTermsTree } from '@wordpress/utils';
```

Comment:
I guess I should have commented on the PR but do you think `buildTermsTree` is useful as a general utility in  `@wordpress/utils` or is it an implementation detail of the `editor` module?

Reply:
Hi @youknowriad, this util is used by latest posts block, and it may be useful to other blocks as well, as blocks should not include editor internals that is the reason why the util is in @wordpress/utils.

Reply:
This feels very much a "utils as a dumping ground of miscellaneous stuff", which I'd like for us to avoid.